src/components/VCarousel/VCarousel.ts

```diff
@@ -31,7 +51,7 @@ export default {
       type: [Number, String],
       default: 6000,
       validator: value => value > 0
-    },
+    } as PropValidator<number>,
```

Comment:
`number | string`?

Reply:
Don't even need the assertion here, primitive types are inferred.